src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -282,7 +282,7 @@ protected function getRedirectRoute(Request $request, $url)
             $this->decodePathInfo($request->getPathInfo()),
             [
                 '_controller' => 'sulu_website.redirect_controller:redirectAction',
-                'url' => $url,
+                'url' => $url . '.' . $request->getRequestFormat(),
```

Comment:
```
$request->getRequestFormat()
```

Will return `html` and `.html` should not be added to the redirect. We need to check for that case.

Reply:
I have implemented a check around html format.